realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -447,6 +446,21 @@ public void singleUserCanBeLoggedInAndOutRepeatedly() {
         }
     }
 
+    // verify that multiple users can be logged in at the same time
```

Comment:
Trying to duplicate a problem that Cartasite is having.  The distinction is whether all of the users are logged in at once, or whether they are logged in serially.  Neither seems to drive the Cartasite bug, so maybe it isn't useful.